pkg/cloudprovider/providers/azure/azure_blobDiskController.go

```diff
@@ -301,17 +297,9 @@ func (c *BlobDiskController) DeleteBlobDisk(diskURI string) error {
 
 //Sets unique strings to be used as accountnames && || blob containers names
 func (c *BlobDiskController) setUniqueStrings() {
-	uniqueString := c.common.resourceGroup + c.common.location + c.common.subscriptionID
-	hash := MakeCRC32(uniqueString)
-	//used to generate a unique container name used by this cluster PVC
-	defaultContainerName = hash
-
-	storageAccountNamePrefix = fmt.Sprintf(storageAccountNameTemplate, hash)
-	// Used to filter relevant accounts (accounts used by shared PVC)
-	storageAccountNameMatch = storageAccountNamePrefix
-	// Used as a template to create new names for relevant accounts
-	storageAccountNamePrefix = storageAccountNamePrefix + "%s"
+	defaultContainerName = MakeCRC32(c.common.resourceGroup + c.common.location + c.common.subscriptionID)
```

Comment:
Any reason we need a unique container name? What about using a fixed one

Reply:
just keep it, it's not related to this PR, I just used the original code.
unique or fixed name, all looks good.